src/ui/control/logo_control.js

```diff
@@ -30,7 +28,7 @@ class LogoControl {
         const anchor = DOM.create('a', 'mapboxgl-ctrl-logo');
         anchor.target = "_blank";
         anchor.rel = "noopener nofollow";
-        anchor.href = "https://www.mapbox.com/";
+        anchor.href = "https://www.maplibre.org/";
```

Comment:
```suggestion
        anchor.href = "https://maplibre.org/";
```

I would use `https://maplibre.org/` instead of `https://www.maplibre.org/` since https://www.maplibre.org/ redirects to https://maplibre.org/ (before redirecting to GitHub).